examples/backend.py

```diff
@@ -99,8 +99,8 @@ def write(db_path: str, table: str, value: Any) -> bool:
                 f"INSERT INTO {table} VALUES (?, datetime('now', 'localtime'))", 
                 (value,)
             )
-    except sqlite3.Error as e:
-        logging.exception("apps.backend.write(): %r", e)
+    except sqlite3.Error:
+        logger.exception("Failed to write database %s from table %s.", db_path, table)
```

Comment:
Since the table is in the database, this expression seems awkward.
How about `"Failed to write a value into table %s of database %s"`?